integrations/rl/puffer/ppo.py

```diff
@@ -713,24 +713,6 @@ def save_checkpoint(data):
     return model_path
 
 
-def try_load_checkpoint(data):
-
-    config = data.config
-    path = os.path.join(config.checkpoint_path, config.exp_id)
-    if not os.path.exists(path):
-        print("No checkpoints found. Assuming new experiment")
-        return
-
-    trainer_path = os.path.join(path, "trainer_state.pt")
-    resume_state = torch.load(trainer_path)
-    data.global_step = resume_state["global_step"]
-    data.epoch = resume_state["update"]
-    model_path = os.path.join(path, resume_state["model_name"])
-    data.uncompiled_policy.load_state_dict(torch.load(model_path).state_dict())
-    data.optimizer.load_state_dict(resume_state["optimizer_state_dict"])
-    print(f'Loaded checkpoint {resume_state["model_name"]}')
-
-
```

Comment:
Unsure how deleting this is relevant to the bug that is fixed but assuming there is a reason

Reply:
Since we are not storing the whole model now (only the state dict), this function cannot directly be used to load a new model. I deleted it to make sure someone doesn't assume it is directly compatible with the current way of storing models